src/main/java/com/acwong/peelplaylistcheck/playlist/PlayListHelperImpl.java

```diff
@@ -0,0 +1,149 @@
+package com.acwong.peelplaylistcheck.playlist;
+
+import com.acwong.peelplaylistcheck.model.*;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.ListIterator;
+
+public class PlayListHelperImpl implements PlayListHelper {
+    private static final String DEFAULT_VIDEO_CONTENT_INFO_FILE = "videolibrary.json";
+
+    private ContentCollection contentCollection;
+
+    /* package */ ContentCollection getContentCollection() {
+        return contentCollection;
+    }
+
+    @Override
+    public boolean loadVideoContentInfo() {
+        return loadVideoContentInfo(DEFAULT_VIDEO_CONTENT_INFO_FILE);
+    }
+
+    @Override
+    public boolean loadVideoContentInfo(String resourceFileName) {
+        ClassLoader classLoader = getClass().getClassLoader();
```

Comment:
Do you really need the ClassLoader? Why not just getClass().getResourceAsStream()?

Reply:
Good suggestion.  Will change to getClass().getResourceAsStream() with prepending '/' to the filename.